mservctl/cmd/update.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/TykTechnologies/mserv/mservclient/client/mw"
+)
+
+// updateCmd represents the update command
+var updateCmd = &cobra.Command{
+	Use:     "update",
+	Short:   "Updates a middleware on mserv",
+	Long:    "Updates a middleware by ID",
+	Example: `mservctl update 13b0eb10-419f-40ef-838d-6d26bb2eeaa8 /path/to/bundle.zip`,
+	Args:    cobra.ExactArgs(2),
+	Run:     updateMiddleware,
+}
+
+func init() {
+	rootCmd.AddCommand(updateCmd)
+}
+
+func updateMiddleware(cmd *cobra.Command, args []string) {
+	file, err := os.Open(args[1])
+	if err != nil {
+		log.WithError(err).Error("Couldn't open the bundle file")
+		return
+	}
+	defer file.Close()
+
+	params := mw.NewMwUpdateParams().WithID(args[0]).WithUploadFile(file)
```

Comment:
Hi @AaronFeledy 
Given your other pull request, https://github.com/TykTechnologies/mserv/pull/26 , where we set the timeout to 2minutes. Would this also require a similar timeout?

Reply:
It would. I have updated it to include a timeout.